doc/radosgw/lua-scripting.rst

```diff
@@ -0,0 +1,308 @@
+=============
+Lua Scripting
+=============
+
+.. versionadded:: Pacific
+
+.. contents::
+
+This feature allows users to upload Lua scripts to different context in the radosgw. The two supported context are "preS3" that will execute a script before the 
+S3 operation was taken, and "postS3" that will execute after each operation is taken. Script may be uploaded to address requests for users of a specific tenant.
+The script can access fields in the request and to modify some fields. All Lua language features can be used in the script.
```

Comment:
I just spotted a typo that I introduced.

Remove the word "to" in line 11:
"The script can access fields in the request and **to** modify some fields."